src/autocompleteInstaller.ts

```diff
@@ -87,11 +86,11 @@ class InstallOptions {
     );
   }
 
-  public static get(testMode: boolean) {
+  public static get() {
```

Comment:
i've added  a mockAutocompleteAPI test function, so there is  not need to check the test mode here